sdk/ai/azure-ai-inference/tests/test_unit_tests.py

```diff
@@ -291,3 +291,91 @@ def test_image_embedding_input_load(self, **kwargs):
             "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAEsAAAApCAYAAAB9ctS7AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsEAAA7BAbiRa+0AAB0CSURBVGhDRZpZzK33ddbXnufh2"
         )
         assert image_embedding_input.text == "some text"
+
+    # **********************************************************************************
+    #
+    #            UNIT TESTS (REGRESSION TESTS) FOR STREAMING RESPONSE PARSING
+    #
+    # **********************************************************************************
+
+    # Recorded from real chat completions streaming response
+    # - Using sample code `samples\sample_chat_completions_streaming.py`,
+    # - With SSE logging turned on (_ENABLE_CLASS_LOGS = True`),
+    # - With Mistral Large model, with user message "how many feet are in a mile?", with content safety turned on.
+    # - The bytes below are from the log output of `[Original element]`.
+    STREAMING_RESPONSE_BYTES: list[bytes] = [
```

Comment:
Why does this need to be so large? I'm assuming that it's not actually comparing to a model response, since it's a unit test? It just seems to overwhelm what else is going on here

Reply:
It has 6 long lines, exactly as returned from the service in one of my runs. I captured it as is. In VSCode I do not have line wrap so it looks much better :-).